.github/workflows/php.yml

```diff
@@ -1,55 +1,52 @@
+---
 name: Unit tests
 
 on:
   push:
+    paths:
+      - '**.php'
+      - 'composer.json'
```

Comment:
That seems unnecessarily restrictive. I can already see a maintainer in a far-distant future (probably me) wonder why a pipeline didn't run after modifying the phpunit.xml, or adding a fixture JSON file, etc. I'm fine with non-code changes triggering an extra (potentially superflous) build for the sake of simplicity.

```suggestion
```

Reply:
That's fine 👍